waspc/data/Generator/templates/react-app/src/webSocket.ts

```diff
@@ -0,0 +1,32 @@
+import { useContext, useEffect } from 'react'
+import { WebSocketContext } from './webSocket/WebSocketProvider';
+
+import type {
+  ClientToServerEvents as _ClientToServerEvents,
+  ServerToClientEvents as _ServerToClientEvents
+} from '../../server/src/webSocket'
```

Comment:
Why are we aliasing the types, and then reassigning them to their original names?

Reply:
Not doing it this way breaks the "import only types from server" magic trick we are using for some reason. Vite starts complaining about *server-side code*

Reply:
I'm not sure if there is something smarter we could be doing to avoid this.

Reply:
It seems that exporting the types from this file directly works as well 👍